storage_service/common/management/commands/create_user.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+
+from tastypie.models import ApiKey
+
+
+class Command(BaseCommand):
+    help = 'Create user with all the arguments specified. ' \
+        'If the user already exists its attributes will be updated.'
+    args = ('username', 'email', 'password', 'api_key')
+
+    def add_arguments(self, parser):
+        parser.add_argument('--username', required=True)
+        parser.add_argument('--email', required=True)
+        parser.add_argument('--password', required=True)
+        parser.add_argument('--api-key', required=True)
+
+    def handle(self, *args, **options):
+        UserModel = get_user_model()
+        user = None
+        try:
+            user = UserModel._default_manager.get(**{UserModel.USERNAME_FIELD: options['username']})
+            print('User found.')
+        except UserModel.DoesNotExist:
+            user = UserModel._default_manager.create_superuser(options['username'], options['email'], options['password'])
```

Comment:
I removed the `using('default')` part of this as it was causing `create_superuser` to fail (it can't be called on a `Queryset`, only a `Manager`). I guess the Jisc work never trigged this (perhaps because we just call this on the test user, and that already exists?). As far as I can tell it's fine to remove that, as it just tells Django to use the default database, and it will do that anyway. I also removed that part from the `get` line too.